src/app/learn/get-started/advanced/accessibility-unit-tests/index.html

```diff
@@ -61,6 +61,14 @@
       expect(element).toBeAccessible();
     }));
   </stache-code-block>
+
+  <p>
+    Keep in mind that <stache-code>async</stache-code> is also a native JavaScript label, so you need to explicitly import the Angular <stache-code>async</stache-code> method at the top of the unit test spec file.
```

Comment:
I would say "native JavaScript declaration,". I know I said "label" on the phone; that wasn't precise enough.